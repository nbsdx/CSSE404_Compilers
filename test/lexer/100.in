// Simple test of comments
/* This should produce completely
/* empty output.


Yep! Totally empty.


*/
