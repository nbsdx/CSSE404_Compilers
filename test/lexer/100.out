ID, Yep
Operator, !
ID, Totally
ID, empty
Delimiter, .
